package.json

```diff
@@ -43,10 +45,10 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "serve": "vite preview",
-    "eslint": "eslint --cache src",
-    "eslint-fix": "eslint --fix --cache src",
-    "prettier": "prettier --list-different '**/*.{js,jsx,md}'",
-    "prettier-fix": "prettier --write '**/*.{js,jsx,md}'",
+    "eslint": "eslint --cache ./src",
+    "eslint-fix": "eslint --fix --cache ./src",
```

Comment:
is this change necessary? Sorry for so many nits :cat:

Reply:
Nope - I'm used to `./` to specify local folder. UNIX? thingy.